pkg/services/sqlstore/sqlstore.go

```diff
@@ -26,6 +27,7 @@ import (
 
 type DatabaseConfig struct {
 	Type, Host, Name, User, Pwd, Path, SslMode string
+	Slaves                                     []string
```

Comment:
Its time we stop using the `master/slave` terminology and use `primary/replica` instead.

Reply:
Done